shared/actions/chat/inbox.js

```diff
@@ -131,7 +131,7 @@ const onInboxStale = function*(): SagaGenerator<any, any> {
       (inbox.items || [])
         .map(c => {
           return new Constants.InboxStateRecord({
-            channelname: c.membersType === ChatTypes.CommonConversationMembersType.team ? ' ' : undefined,
+            channelname: c.membersType === ChatTypes.CommonConversationMembersType.team ? '-' : undefined,
```

Comment:
we don't have the team name yet as its not part of the untrusted call, yet its still a channel so we need a key but dont' know the value